admin/weaviate/permissions.py

```diff
@@ -0,0 +1,365 @@
+#!/usr/bin/env python3
+"""
+Weaviate RBAC Setup for Ed# User configurations for each microservice (email format required by Weaviate)
+MICROSERVICE_USERS = {
+    "atlas": {
+        "username": "atlas-service@edutelligence.local",
+        "description": "Atlas microservice - Exercise and competency management"
+    },
+    "iris": {
+        "username": "iris-service@edutelligence.local", 
+        "description": "Iris microservice - FAQ and lecture unit management"
+    },
+    "athena": {
+        "username": "athena-service@edutelligence.local",
+        "description": "Athena microservice - Assessment and programming analysis"
+    }
+}roservices
+
+This script sets up Role-Based Access Control (RBAC) for Weaviate to allow
+each microservice (Atlas, Iris, Athena) to manage their own collections
+on a shared Weaviate instance while preventing interference with other services.
+
+Usage:
+    python permissions.py --host localhost --port 8080 --create-users --create-roles --assign-permissions
+
+Requirements:
+    - Weaviate instance with RBAC enabled
+    - Admin credentials for initial setup
+"""
+
+import argparse
+import logging
+import sys
+from typing import List, Dict, Any
+import weaviate
+
+
+# Configure logging
+logging.basicConfig(level=logging.INFO, format='%(asctime)s - %(levelname)s - %(message)s')
+logger = logging.getLogger(__name__)
+
+
+# Collection prefix mappings for each microservice
+# Each service can access collections that start with their prefix
+MICROSERVICE_PREFIXES = {
+    "atlas": "atlas_",
+    "iris": "iris_", 
+    "athena": "athena_"
+}# User configurations for each microservice
+# Note: Weaviate RBAC requires email format for usernames, but these are service accounts
+MICROSERVICE_USERS = {
+    "atlas": {
+        "username": "atlas-service@edutelligence.local",
+        "password": "atlas_secure_password_2024",  # Should be changed in production
+    },
+    "iris": {
+        "username": "iris-service@edutelligence.local", 
+        "password": "iris_secure_password_2024",   # Should be changed in production
+    },
+    "athena": {
+        "username": "athena-service@edutelligence.local",
+        "password": "athena_secure_password_2024", # Should be changed in production  
+    }
+}
```

Comment:
Why are you defining this at all? This should be created solely from the values in the environment, no?